app/controllers/api/v1/books_controller.rb

```diff
@@ -5,17 +5,25 @@ module V1
     # BooksController
     class BooksController < ApplicationController
       def index
-        render json: Book.all.order(id: :asc)
+        @books = Book.paginate_results(params[:page])
       end
 
       def create
-        book = Book.create(book_params)
-        render json: book
+        @book = Book.new(book_params)
+        if @book.save
+          render 'api/v1/books/create'
```

Comment:
When we are rendering from views, we don't need to type in the file path. Since the controller will already know which file we need:

```ruby
def create
 @book = Book.new(book_params)
 if @book.save
  render :create
 else
  render :error
 end
end
```